test/Microsoft.ML.Tests/DatabaseLoaderTests.cs

```diff
@@ -27,6 +27,8 @@ public DatabaseLoaderTests(ITestOutputHelper output)
         public void IrisLightGbm()
         {
             var mlContext = new MLContext(seed: 1);
+            var connectionString = GetDataPath(TestDatasets.iris.trainFilename);
+            var commandText = "Label;SepalLength;SepalWidth;PetalLength;PetalWidth";
```

Comment:
Why is the command text (`commandText`) having just a list of fields/columns? What if the user wants to run a join query against multiple tables?
Also, where the name of the database **table** would be specified in the sample if you are not accepting a real SQL statement?
I might be missing some context here.. sorry..

Reply:
This is just for the MockCommand for testing purposes and is being done because we dont have a good way of hosting a database for the tests.

Reply:
The MockCommand/Connection/Reader are themselves just a thin shim over the TextLoader